parm/config/gefs/config.nsst

```diff
@@ -12,10 +12,6 @@ export NST_MODEL=2
 
 # nstf_name(2) : NST_SPINUP : 0 = OFF, 1 = ON,
 export NST_SPINUP=0
-cdate="${PDY}${cyc}"
-if (( cdate < 2017072000 )); then
-    export NST_SPINUP=1
-fi
```

Comment:
Curious why this is being removed? No longer need?

Reply:
The ICs used for the reforecasting effort have NSST spun up.